llama_stack/providers/inline/agents/meta_reference/persistence.py

```diff
@@ -21,6 +21,7 @@
 class AgentSessionInfo(BaseModel):
     session_id: str
     session_name: str
+    # TODO: is this used anywhere?
```

Comment:
Maybe remove this `vector_db_id` within the Agent session? Should be possible for an agent session to have access to multiple vector_db_ids?

Reply:
Seems arbitrary to have a single vector db id here. Maybe remove in another PR?